apps/condo/domains/common/components/Tabs.tsx

```diff
@@ -70,27 +70,35 @@ const TabWrapper = styled.div`
   }
 `
 
-interface TabProps extends ITrackingComponent {
+interface TabProps {
     title: string
+    tabKey: string
     showIcon?: boolean
     icon?: React.ReactNode
     onClick?: React.MouseEventHandler<HTMLDivElement>
 }
 
 export const Tab: React.FC<TabProps> = ({
     title,
+    tabKey,
     showIcon = true,
     icon = <StarIcon/>,
-    eventName: propEventName,
     onClick,
 }) => {
-    const { getTrackingWrappedCallback, getEventName } = useTracking()
+    const onClickWrapped: React.MouseEventHandler<HTMLDivElement> = useCallback((e) => {
+        analytics.track('change', {
+            component: 'Tabs',
+            location: window.location.href,
```

Comment:
Is it possible to transfer the page address automatically?

Reply:
It's kinda basic info, that could be attached to the event. So for me it's strange that dev needs to pass location every time via constant

Reply:
It's legacy version of components here. 

All ui-kit components pass location automatically, so I add this to be compatible with UI-kit ones. The goal is to remove this component entirely